pyproject.toml

```
@@ -22,7 +22,7 @@ pretty = true
 scripts_are_modules = true
 show_error_codes = true
 show_error_context = true
-exclude = "(vendor*|venv*)/$"
+exclude = ["vendor/"]
```

I don't think we want to remove `venv` ; I assume it otherwise will try to scan through the virtualenv directory, and become very slow. Was there a reason you removed it?

```suggestion
exclude = ["vendor/", "venv/"]
```

---

Sgtm!